### DISCUSSION TITLE
New Miner Around 300 sectors but got Large kvlog/metadata (Resolved)

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#8324] [General Q&A] [answered]
I created new miner several days ago ...

but the new miner folder (root/.lotusminer)kvlog/metadata already 290G ????
Why so large ??
And how to resolve ??
Can delete ?

**Comment by seantw1974** [comment]
Seems can Delete and Restart Miner .

**Comment by TippyFlitsUK** [comment, marked as answer]
Many thanks for the update @seantw1974 .

You can also disable the metadata log altogether by specifying `DisableMetadataLog = true` in your miner `config.toml.`

[Please be sure to take regular miner backups!](https://lotus.filecoin.io/storage-providers/operate/backup-and-restore/)

**Reply by seantw1974** [reply]
Yes, already modify it !
Thank you!
